features/breol_news/breol_news.module

```diff
@@ -19,3 +19,22 @@ function breol_news_node_validate($node, $form, &$form_state) {
     }
   }
 }
+
+/**
+ * Implements hook_views_query_alter().
+ *
+ * Makes the "article" view show nodes of type "breol_news".
```

Comment:
Maybe more information about why this is not just done in the views UI and added to the feature the view comes from?

Reply:
I've added a comment on this.